src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String DOTTED_LINE = "\t-----------------------------------------";
+    private static final int MAX_TASKS = 100;
+    private static final String EXIT_COMMAND = "bye";
+    private static final String LIST_COMMAND = "list";
+    private static final String DONE_COMMAND = "done";
+    private static final String TODO_COMMAND = "todo";
+    private static final String DEADLINE_COMMAND = "deadline";
+    private static final String EVENT_COMMAND = "event";
+    private static final String BY_COMMAND = "/by";
+    private static final String AT_COMMAND = "/at";
+
+    private static final Task[] tasks = new Task[MAX_TASKS];
+    private static int numOfTasks = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+
+        Scanner in = new Scanner(System.in);
+        String input = in.nextLine();
+
+        while (!input.equalsIgnoreCase(EXIT_COMMAND)) {
+            System.out.println(DOTTED_LINE);
+            if (input.equalsIgnoreCase(LIST_COMMAND)) {
+                handleInput(input, LIST_COMMAND);
+            } else if (input.startsWith(DONE_COMMAND)) {
+                handleInput(input, DONE_COMMAND);
+            } else if (input.startsWith(TODO_COMMAND)) {
+                handleInput(input, TODO_COMMAND);
+            } else if (input.startsWith(DEADLINE_COMMAND)) {
+                handleInput(input, DEADLINE_COMMAND);
+            } else if (input.startsWith(EVENT_COMMAND)) {
+                handleInput(input, EVENT_COMMAND);
+            } else {
+                handleInput(input, "");
+            }
+            System.out.println(DOTTED_LINE);
+            input = in.nextLine();
+        }
+
+        printEndMessage();
+    }
+
+    private static void printWelcomeMessage() {
+        System.out.println(LOGO);
+        System.out.println(DOTTED_LINE);
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println(DOTTED_LINE);
+    }
+
+    private static void handleInput(String input, String command) {
+        switch (command) {
+        case LIST_COMMAND: {
```

Comment:
Should the opening brace after case LIST_COMMAND: and the closing brace after that be removed to follow coding standards?